resources/kcp/charts/kyma-environment-broker/files/swagger.yaml

```diff
@@ -58,6 +58,24 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/StatusResponseList'
+    delete:
+      summary: Cancels in progress orchestration
+      operationId: cancelDynamic
+      description: |
+        Cancels in progress orchestration depends on their state from storage
```

Comment:
how about
```
Cancels all orchestration in progress
```

Reply:
I will go with Cancels in progress orchestration